<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer</class>
 <widget class="QWidget" name="ImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ImageViewer{background-color:transparent;}</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>1024</width>
     <height>640</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1022</width>
      <height>638</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="m_imageLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="m_closeImageButton">
   <property name="geometry">
    <rect>
     <x>1114</x>
     <y>5</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-image:url(:/images/filewidgetimage/btn_X_nor.png)} 
QPushButton { border-style: outset}
QPushButton:pressed {background-image:url(:/images/filewidgetimage/btn_X_nor_click.png)} QPushButton:pressed { border-style: outset};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_nextButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>685</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_preButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>685</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;Previous</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QSlider" name="m_scaledSlider">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>685</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="m_imageNameLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_scaledlabel">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>685</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="m_zoomInButton">
   <property name="geometry">
    <rect>
     <x>1094</x>
     <y>685</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_zoomOutButton">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>685</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
